client/src/utils/user-full-name.ts

```diff
@@ -0,0 +1,13 @@
+import type { User } from 'types';
+
+const getUserFullName = (user: User, replaceByEmail = false) => {
```

Comment:
Instead of a `replaceByEmail = false` parmeter, I would suggest something like an `options` object. Why? Well, if in the future there are more options to contemplate, with the current approach, you will need to add more parameters to the function, meaning updating the function where is used and forcing to specific all values eventually.

If you have something like 
```
options = {
  replaceByEmail: false,
  useOnlyFirstName: true,
  someRandomFutureOption: false,
},

// your logic
if (options.replaceByEmail) //...
if (options.useOnlyFirstName) //...
if (options.someRandomFutureOption) //...
```

This way you only need to work extending the business logic of the function and forget about updating parameters across the codebase.

TLDR: the code si completely fine, it's just a matter of scalability

Reply:
Also, don't forget to type the return of the function with `:string` :)

Reply:
Also (and I finish, for real), I think is more useful for you, and other developers, knowing the API of your function is `getUserFullName(user, { replaceByEmail: true })` instead of `getUserFullName(user, true)` (as you don't what that `true` really means and would need to check on the function)

Reply:
Great review! Thanks!